themes/tiddlywiki/vanilla/base.tid

```diff
@@ -135,6 +135,10 @@ input:not([type]) {
 	background: <<colour background>>;
 }
 
+input[type="checkbox"] {
+  vertical-align: middle;
+}
+
```

Comment:
This setting is generic and should be added to the core, independent to this PR.